static/js/cluster-stats.js

```diff
@@ -573,31 +573,54 @@ function processClusterStats(res) {
         ],
     });
 
+    function formatIngestVolume(volume) {
+        let volumeGB;
+    
+        if (typeof volume === 'string') {
+            // Remove " GB" if it's in the string
+            volumeGB = parseFloat(volume.replace(" GB", ""));
+        } else if (typeof volume === 'number') {
+            volumeGB = volume;
+        } else {
+            // Handle unexpected input types
+            console.error("Unexpected volume type:", typeof volume);
+            return "N/A";
+        }
+    
+        if (isNaN(volumeGB)) {
+            console.error("Invalid volume value:", volume);
+            return "N/A";
+        }
+        
+        if (volumeGB === 0) {
+            return '0 GB';
+        } else if (volumeGB < 1) {
+            return '< 1 GB';
+        } else {
+            return `${Math.floor(volumeGB).toLocaleString('en-US')} GB`;
```

Comment:
use Math.round instead of math.floor , so e.g.4.5 will be shown as 5 instead of 4